build.xml

```diff
@@ -1,126 +1,13 @@
-  </target>
-  <target name="compileTest" depends="compile,compileMock">
-    <javac srcdir="${test.src}" destdir="${test.build}" includeantruntime="false">
-      <classpath>
-        <pathelement path="${build}"/>
-        <pathelement location="/usr/share/java/junit4.jar"/>
-        <fileset dir="${test.lib}">
-          <include name="*.jar"/>
-        </fileset>
-      </classpath>
-    </javac>
-  </target>
-  <target name="test" description="run tests" depends="compileTest">
-    <junit printSummary="no" fork="yes" failureproperty="junit.failure">
-      <classpath>
-        <pathelement location="${test.build}"/>
-        <pathelement path="${build}"/>
-        <pathelement location="/usr/share/java/junit4.jar"/>
-        <fileset dir="${test.lib}">
-          <include name="*.jar"/>
-        </fileset>
-      </classpath>
-      <formatter type="plain" usefile="false" />
-      <formatter type="plain" />
-      <batchtest todir="${test.build}/output">
-        <fileset dir="${test.src}" includes="**/*.java" />
-      </batchtest>
-    </junit>
-    <fail if="junit.failure" message="Unit test(s) failed.  See reports!"/>
-  </target>
-  <target name="coverage" description="run tests with coverage" depends="compileTest">
-    <taskdef uri="antlib:org.jacoco.ant" resource="org/jacoco/ant/antlib.xml">
-      <classpath>
-        <fileset dir="${test.lib}">
-          <include name="*.jar"/>
-        </fileset>
-      </classpath>
-    </taskdef>
-    <jacoco:coverage>
-      <junit printSummary="no" fork="yes" failureproperty="junit.failure">
-        <classpath>
-          <pathelement location="${test.build}"/>
-          <pathelement path="${build}"/>
-          <pathelement location="/usr/share/java/junit4.jar"/>
-          <fileset dir="${test.lib}">
-            <include name="*.jar"/>
-          </fileset>
-        </classpath>
-        <formatter type="plain" usefile="false" />
-        <formatter type="plain" />
-        <batchtest todir="${test.build}/output">
-          <fileset dir="${test.src}" includes="**/*.java" />
-        </batchtest>
-      </junit>
-    </jacoco:coverage>
-    <jacoco:report>
-      <executiondata>
-        <file file="jacoco.exec"/>
-      </executiondata>
-      <structure name="ykneo-openpgp project">
-        <classfiles>
-          <fileset dir="applet/bin"/>
-        </classfiles>
-        <sourcefiles encoding="UTF-8">
-          <fileset dir="applet/src"/>
-        </sourcefiles>
-      </structure>
-      <xml destfile="jacoco.xml"/>
-      <html destdir="jacoco"/>
-    </jacoco:report>
-    <fail if="junit.failure" message="Unit test(s) failed.  See reports!"/>
+  <taskdef name="javacard" classname="pro.javacard.ant.JavaCard" classpath="ant-javacard.jar"/>
+  <target name="applet">
+    <javacard>
+      <cap aid="A0:00:00:06:17:00:EC:5E:FF:D8" output="PGP.cap" sources="applet/src">
+        <applet class="openpgpcard.OpenPGPApplet" aid="A0:00:00:06:17:00:EC:5E:FF:D8:00:00:00:01:00:00"/>
```

Comment:
You are changing the default applet AID. Is this intentional? IIRC gpg is picky about AID-s